src/components/home/elements/SendToken.vue

```diff
@@ -527,38 +536,29 @@ export default {
         },
         sendData: function(walletType) {
             var apiSchema
-            console.log('addresses: ' + JSON.stringify(this.addresses))
             if (walletType === 'hotWallet') {
                 if (this.hasConfirmed) {
                     return
                 }
                 this.hasConfirmed = true
-                this.contractId = transaction.tokenIDToContractID(this.tokenId)
-                this.fee = BigNumber(CONTRACT_EXEC_FEE * VSYS_PRECISION)
                 this.feeScale = 100
                 const dataInfo = {
                     contractId: this.contractId,
                     senderPublicKey: this.getKeypair(this.addresses[this.address]).publicKey,
                     fee: CONTRACT_EXEC_FEE * VSYS_PRECISION,
                     feeScale: FEE_SCALE,
                     timestamp: this.timeStamp,
-                    description: this.attachment,
-                    funcIdx: SEND_FUNCIDX,
-                    data: transaction.prepareSend(this.recipient, BigNumber(this.amount)),
-                    signature: transaction.prepareExecContractSignature(this.contractId, SEND_FUNCIDX, transaction.prepareSend(this.recipient, BigNumber(this.amount)), this.attachment, BigNumber(this.fee), this.feeScale, BigNumber(this.timeStamp), this.getKeypair(this.addresses[this.address]).privateKey)
+                    attachment: '',
```

Comment:
The attachment is not pass to JSON when call execute contract API. This will cause invalid signature error if there is attachment input.